tests/backup/backup_resiliency_test.go

```diff
@@ -1299,8 +1308,8 @@ var _ = Describe("{RebootNodesWhenBackupsAreInProgress}", func() {
 		log.Infof("Switching cluster context to destination cluster")
 		err := SetDestinationKubeConfig()
 		log.FailOnError(err, "Switching context to destination cluster failed")
-		listOfWorkerNodes = node.GetWorkerNodes()
-		for _, node := range listOfWorkerNodes {
+		listOfStorageDriverNodes = node.GetStorageDriverNodes()
```

Comment:
Since listOfStorageDriverNodes is a global variable at the test case level, there is no need to retrieve the nodes again.

Reply:
done